dss-api.yml

```diff
@@ -1,7 +1,47 @@
 swagger: '2.0'
 info:
   title: HCA DCP DSS API
-  description: Human Cell Atlas Data Coordination Platform Data Storage System API
+  description: |
+    Human Cell Atlas Data Coordination Platform Data Storage System API
+
+    **HTTP semantics**
+
+    The DSS API requires clients to follow certain HTTP protocol semantics that may require extra configuration in your
+    HTTP client. The reference CLI and SDK (https://hca.readthedocs.io/) is pre-configured to do this. If writing your own
+    client, please note the following:
+    
+    **301 redirects**: Some DSS API routes may return one or more HTTP 301 redirects, including potentially redirects to
+    themselves (combined with the **Retry-After** delay described below). The client must follow these redirects to obtain
+    the resource requested.
+
+    **Retry-After header**: Some DSS API routes may use the **Retry-After** header in combination with HTTP 301 or 500
```

Comment:
I think we only return Retry-After with HTTP 301.

Reply:
I would like to reserve the right to return Retry-After with 5xx in the future :)